### PR TITLE
Update functions to merge transactions properly

### PR DESCRIPTION
**Pull request by kchulj**
The `mergeTx` function was creating the same number of `Output`s as `Input`s so it was not actually merging.
We decide to either split or merge based on the number of `UTXO`, and these functions adjust the number. Merging should decrease the number of `UTXO` by definition as below :

```
[main(J2TR) INF] UTXO delta: -15
```
The most basic way (current implementation) is to create a single `Transaction` with multiple `Input`s and a single `Output`.

Also, `filter` should have been called in `splitTx` instead of `mergeTx`. Its role is to skip splitting `Output`s with `Amount` less than `count` so we don't end up with `Amount(0)`.

### CODE REVIEW

File: source/faucet/main.d [hunk 1]
```diff
@@ -106,6 +106,7 @@ private auto splitTx (UR) (UR utxo_rng, uint count)
     static assert (isInputRange!UR);
 
     return utxo_rng
+        .filter!(tup => tup.value.output.value >= Amount(count))
```
Comment (TrustHenry) [inline source/faucet/main.d:109]:
```suggestion
        .filter!(tup => tup.value.output.value >= Amount(count))
```
count?

Comment (kchulj) [reply source/faucet/main.d:109]: Oops.

Comment (kchulj) [reply source/faucet/main.d:109]: > count?

When `splitTx` is called, it takes the number from `Config.count`. It is the number of keys that the `Amount` is split among. However, if we have an `Amount` that is less than `count`, we get an `Output` with `Amount(0)` This triggers `assert(0)`.

https://github.com/bpfkorea/agora/blob/3402716cf39f69b31995ba205730bd189aa761ab/source/agora/utils/Test.d#L583

**Comment by kchulj** [comment]
Ready for review.

File: source/faucet/main.d [hunk 2]
```diff
@@ -41,7 +41,7 @@ import vibe.web.rest;
 private struct Config
 {
     /// How frequently we run our periodic task
-    static immutable interval = 15.seconds;
+    static immutable interval = 30.seconds;
```
Comment (AndrejMitrovic) [inline source/faucet/main.d:44]: This should be moved into its own commit and explained why the change was made.

**Review by AndrejMitrovic — CHANGES_REQUESTED** [review]
Just one comment.

File: source/faucet/main.d (hunk 2 above)
Comment (kchulj) [reply source/faucet/main.d:44]: Fixed

**Review by AndrejMitrovic — APPROVED** [review]
